docker-compose-agent-ipfs.yml

```diff
@@ -55,6 +55,21 @@ services:
     networks:
       - testnet
 
+  ##################################################################################
+  ## OPA ###########################################################################
+  ##################################################################################
+
+  opa:
+    container_name: opa
+    image: openpolicyagent/opa:0.60.0-static
+    ports:
+      - '8181:8181'
```

Comment:
Do we want this exposed outside the container?

Reply:
I guess not in the real world. It's useful for developing against to be able to send HTTP requests directly to OPA